### PR TITLE
Updated ClearArt paths and Aspect Ration logo Xpos for resized logos

### PR DESCRIPTION
**Pull request by KJ-Cox**
posX change for resized aspect ratio icon in movingpictures.mediainfo.xml, movingpictures.mediainfo.details.xml, myvideo.mediainfo.xml and DialogVideoInfo.xml.

videoFullScreen.xml added FH paths for the video fullscreen pause ClearArt.
common.overlay.xml added FH paths for the video playing GUI video spinning CDArt.
videoOSD.xml I am not sure where it comes into play but I updated the values for ClearArt.

### CODE REVIEW

File: Common/DialogVideoInfo.xml
Comment (andrewjswan) [inline Common/DialogVideoInfo.xml:1]: What has changed here? I don’t see the history; it’s difficult to understand, at first glance, how everything was.

File: Common/common.overlay.xml
Comment (andrewjswan) [inline Common/common.overlay.xml:1]: What has changed here? I don’t see the history; it’s difficult to understand, at first glance, how everything was.

File: Common/myvideo.mediainfo.xml
Comment (andrewjswan) [inline Common/myvideo.mediainfo.xml:1]: What has changed here? I don’t see the history; it’s difficult to understand, at first glance, how everything was.

**Comment by andrewjswan** [comment]
Can you describe in a nutshell what has changed in the newly added files compared to the Titan skin? I don’t see the history; it’s difficult to understand, at first glance, how everything was.

File: Common/DialogVideoInfo.xml
Comment (KJ-Cox) [reply Common/DialogVideoInfo.xml:1]: aspect ratio icon posX value because the logo was resized and centered

File: Common/common.overlay.xml
Comment (KJ-Cox) [reply Common/common.overlay.xml:1]: aspect ratio icon posX value because the logo was resized and centered

Comment (KJ-Cox) [reply Common/common.overlay.xml:1]: Changed value to #fanarthandler.movie.cd.selected for <description>Movie CDArt</description>

File: Common/DialogVideoInfo.xml
Comment (KJ-Cox) [reply Common/DialogVideoInfo.xml:1]: Look for <description>Aspect ratio</description> and <description>AspectRatio</description>

File: Common/myvideo.mediainfo.xml
Comment (KJ-Cox) [reply Common/myvideo.mediainfo.xml:1]: aspect ratio icon posX value because the AspectRatio logos were resized and centered,

**Comment by KJ-Cox** [comment]
> Can you describe in a nutshell what has changed in the newly added files compared to the Titan skin? I don’t see the history; it’s difficult to understand, at first glance, how everything was.

FanartHandler paths used instead of relatative paths for the video fullscreen pause ClearArt and video playing GUI video spinning CDArt.

Other files have the medi info aspect ratio icon moved to the right now that I resized and centered the images.

**Pull request merged**